debian/scripts/usr/local/bin/config_save

```diff
@@ -0,0 +1,36 @@
+# Save current configurations to partition containing conf.d
+#
+# Copyright (C) 2018 Benedict Lau <benedict.lau@groundupworks.com>
+
+CONFDIR=conf.d
+
+try_partition() {
+    # Create file system node from partition
+    #mknod /dev/$1 b $2 $3 2>/dev/null
+
+    # Mount partition as read-write
+    mount /dev/$1 /mnt
+
+    # Find conf.d in root of partition and save current configurations in /etc
+    if [ -d "/mnt/$CONFDIR" ]; then
+        echo Saving configurations to /dev/$1: "/mnt/$CONFDIR/50-node-config-save.tar.gz"
+        # TODO Handle symlinks properly (i.e. excluding them from the tar)
```

Comment:
Right now it tars up everything in /etc and when extracting the tar you end up with a bunch of symlink errors like this:

```
tar: ./network/if-post-down.d/hostapd: Cannot create symlink to '../../hostapd/ifupdown.sh': Operation not permitted
```

 I guess we should exclude symlinks in the tar?

Reply:
@hamishcoleman this is the biggest problem though

Reply:
cannot exclude symlinks - they are vital to be stored as part of the config.

Dont know why you get untar errors - tar quite happily handles creating symlinks

Reply:
Not sure, but I was also runnings the script not in the expected place